src/components/tasks/CreateTasks.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useRef, useState } from "react";
+import { useDispatch } from "react-redux";
+import { addTasks } from "src/store/actions/taskActions";
+import "src/components/tasks/CreateTasks.component.scss";
+import { createDate } from "src/utils/helpers/createDate";
+import PropTypes from "prop-types";
+
+const CreateTasks = ({ show, onShow }) => {
+  const dispatch = useDispatch();
+  const [task, setTask] = useState("");
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    inputRef.current.focus();
+  }, []);
+
+  const handleChange = (e) => {
+    setTask(e.target.value);
+  };
+
+  const add = () => {
+    if (task === "") {
+      return;
+    } else {
+      dispatch(
+        addTasks({
+          content: task,
+          createdAt: createDate(),
+        })
+      );
+      setTask("");
+      onShow(!show);
+    }
+  };
+
+  return (
+    <div className="addTasks">
+      <input
+        type="text"
+        onChange={(e) => handleChange(e)}
```

Comment:
instead of calling function, please pass instance.
```suggestion
        onChange={handleChange}
```